consensus/src/chained_bft/chained_bft_smr.rs

```diff
@@ -98,53 +89,14 @@ impl<T: Payload> ChainedBftSMR<T> {
         self.block_store.clone()
     }
 
-    fn create_pacemaker(
-        &self,
-        time_service: Arc<dyn TimeService>,
-        timeout_sender: channel::Sender<Round>,
-    ) -> Pacemaker {
-        // 1.5^6 ~= 11
-        // Timeout goes from initial_timeout to initial_timeout*11 in 6 steps
-        let time_interval = Box::new(ExponentialTimeInterval::new(
-            self.config.pacemaker_initial_timeout,
-            1.5,
-            6,
-        ));
-        Pacemaker::new(time_interval, time_service, timeout_sender)
-    }
-
-    /// Create a proposer election handler based on proposers
-    fn create_proposer_election(
-        &self,
-        validators: &ValidatorVerifier,
-    ) -> Box<dyn ProposerElection<T> + Send + Sync> {
-        let proposers = validators.get_ordered_account_addresses();
-        match self.config.proposer_type {
-            ConsensusProposerType::MultipleOrderedProposers => {
-                Box::new(MultiProposer::new(proposers, 2))
-            }
-            ConsensusProposerType::RotatingProposer => Box::new(RotatingProposer::new(
-                proposers,
-                self.config.contiguous_rounds,
-            )),
-            // We don't really have a fixed proposer!
-            ConsensusProposerType::FixedProposer => {
-                let proposer = choose_leader(proposers);
-                Box::new(RotatingProposer::new(
-                    vec![proposer],
-                    self.config.contiguous_rounds,
-                ))
-            }
-        }
-    }
-
     fn start_event_processing(
         executor: TaskExecutor,
+        _epoch_mgr: EpochManager<T>,
```

Comment:
names beginning with _ usually imply unused.

if it is unused can we remove it from here?